inskin-media/inskin-media-htb.js

```diff
@@ -0,0 +1,564 @@
+         *        slots: [
+         *             {
+         *                 placementId: 54321,
+         *                 sizes: [[300, 250]]
+         *             },{
+         *                 placementId: 12345,
+         *                 sizes: [[300, 600]]
+         *             },{
+         *                 placementId: 654321,
+         *                 sizes: [[728, 90]]
+         *             }
+         *         ],
+         *         site: 'http://google.com'
+         *     },
+         *     callbackId: '_23sd2ij4i1' //unique id used for pairing requests and responses
+         * }
+         */
+
+        /* ---------------------- PUT CODE HERE ------------------------------------ */
+        var data = {
+            time: System.now(),
+            user: {},
+            url: Browser.getPageUrl(),
+            referrer: Browser.getReferrer(),
+            enableBotFiltering: true,
+            includePricingData: true
+        };
+
+        /* Change this to your bidder endpoint. */
+        var baseUrl = Browser.getProtocol() + '//mfad.inskinad.com/api/v2';
+
+        /* ------------------------ Get consent information -------------------------
+         * If you want to implement GDPR consent in your adapter, use the function
+         * ComplianceService.gdpr.getConsent() which will return an object.
+         *
+         * Here is what the values in that object mean:
+         *      - applies: the boolean value indicating if the request is subject to
+         *      GDPR regulations
+         *      - consentString: the consent string developed by GDPR Consent Working
+         *      Group under the auspices of IAB Europe
+         *
+         * The return object should look something like this:
+         * {
+         *      applies: true,
+         *      consentString: "BOQ7WlgOQ7WlgABABwAAABJOACgACAAQABA"
+         * }
+         *
+         * You can also determine whether or not the publisher has enabled privacy
+         * features in their wrapper by querying ComplianceService.isPrivacyEnabled().
+         *
+         * This function will return a boolean, which indicates whether the wrapper's
+         * privacy features are on (true) or off (false). If they are off, the values
+         * returned from gdpr.getConsent() are safe defaults and no attempt has been
+         * made by the wrapper to contact a Consent Management Platform.
+         */
+        var gdprStatus = ComplianceService.gdpr.getConsent();
+
+        if (gdprStatus.applies) {
+            data.consent = {
+                gdprVendorId: 150,
+                gdprConsentString: gdprStatus.consentString,
+                gdprConsentRequired: true
+            };
+        }
+
+        data.placements = returnParcels.map(function (parcel) {
+            var placement = {
+                networkId: configs.networkId,
+                siteId: configs.siteId,
+                divName: parcel.xSlotName,
```

Comment:
The `divName` property is being set to the parcel's `xSlotName`. This is not actually the `divId` and not guaranteed to be stable (i.e. it can change at any time). So if this is not what is expected, it should not be used.

Reply:
This is fine, this is just the key used by the ad server for requests/responses (see example in Documentation) - does not need to be stable